README.md

```diff
@@ -72,5 +72,26 @@ hexdump -n 32 -e '8 "%08x" 1 "\n"' /dev/random
 od -vN 32 -An -tx1 /dev/random | tr -d " \n" ; echo
 ```
 
+## Config
+
+In order to run with config, run with `--config <file>`.
+Also you can override settings of config file with command line arguments.
+Example config.json is below.
+```$xslt
+{
```

Comment:
Sorry, but there are more useful formats for Rust like RSON. @kurnevsky which one do you like?

Reply:
Actually I like yaml. It's not so verbose (no need to write brackets) and also widely used for configs (alacrytty, for instance). Serde supports it via https://github.com/dtolnay/serde-yaml

Reply:
RSON or RON is supported by serve, but these are not supported by config-rs.
https://docs.rs/config/0.9.1/config/enum.FileFormat.html